packages/apps/human-app/frontend/src/shared/utils/time.ts

```diff
@@ -0,0 +1,22 @@
+export type ProposalStatus = 'pending' | 'active';
+
+export function getProposalStatus(
+  voteStartMs: number,
+  voteEndMs: number,
+  now: number = Date.now()
```

Comment:
```suggestion
  proposal: Proposal
```
declare `now` inside and use as `proposal.voteStart <= now`, because `now` is part of this function logic and you don't have to pass it as argument.

Also passing just a proposal is simpler for the caller of this util